src/components/BrowserQueryMenu/BrowserQueryMenu.tsx

```diff
@@ -41,6 +45,8 @@ const BrowserQueryMenu: FC<Props> = ({ onEditQuery, onRemoveQuery }) => {
   const [actionsMenu, setActionsMenuVisibility] = useState(false);
   const [actionsTooltip, showActionsTooltip] = useState(false);
   const [settingsTooltip, showSettingsTooltip] = useState(false);
+  const [shareTooltip, showShareTooltip] = useState(false);
```

Comment:
what do you think about refactoring these `useStates` into one?

Reply:
e.g `const [visibile, setVisible] = useState<'menu' | 'actions' | 'settings' | 'share'>(false)`

Reply:
Thanks, good idea to refactor this. I changed menu tooltips to use MousePositionedTooltips component, so now it looks  much simpler.